<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SushiClass</class>
 <widget class="QWidget" name="SushiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sushi</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_totle_budget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>总预算</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入：寿司的价格/等级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>寿司种类数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Sushi_num">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>结果</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="LogOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>431</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Sushi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SushiClass</receiver>
   <slot>Calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LogOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Sushi_num</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_totle_budget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Calculate()</slot>
 </slots>
</ui>
